tests/test_rdpg_corr.py

```diff
@@ -79,5 +79,5 @@ def test_p_is_close(self):
         graphs2 = np.stack(graphs2)
         np.testing.assert_allclose(np.mean(graphs1, axis=0), P, atol=0.1)
         np.testing.assert_allclose(np.mean(graphs2, axis=0), P, atol=0.1)
-        np.testing.assert_allclose(np.mean(np.mean(graphs1, axis=0) - P), 0, atol=1e-4)
-        np.testing.assert_allclose(np.mean(np.mean(graphs2, axis=0) - P), 0, atol=1e-4)
+        # np.testing.assert_allclose(np.mean(np.mean(graphs1, axis=0) - P), 0, atol=1e-4)
+        # np.testing.assert_allclose(np.mean(np.mean(graphs2, axis=0) - P), 0, atol=1e-4)
```

Comment:
im okay with just deleting these lines for good at this point

Reply:
ok will do!

Reply:
https://github.com/microsoft/graspologic/pull/541/commits/857bc0241a257b838611e876b019d7186680ebb9